src/kili/entrypoints/mutations/label/__init__.py

```diff
@@ -159,17 +161,22 @@ def append_to_labels(
             user = self.get_user()  # type: ignore  # pylint: disable=no-member
             author_id = user["id"]
 
+        # fmt: off
```

Comment:
why is it needed?

Reply:
oops not needed anymore, it was not playing well with the type: ignore at the time. Fixing it